go/cmd/vtctldclient/command/keyspaces.go

```diff
@@ -27,7 +27,7 @@ import (
 	"vitess.io/vitess/go/cmd/vtctldclient/cli"
 	"vitess.io/vitess/go/mysql"
 	"vitess.io/vitess/go/vt/logutil"
-	"vitess.io/vitess/go/vt/sidecardb"
+	"vitess.io/vitess/go/vt/sidecardb/dbname"
```

Comment:
I would have preferred us to move this up versus down. That is, something like `go/vt/constants` or even `go/constants` versus embedding it deeper in. Was there a reason you chose this approach?

Reply:
Done